15-components-lab_v16/src/01-timeconverter.jsx

```diff
@@ -8,5 +8,38 @@
 
 import React from 'react';
 
-export default React.createClass({
-});
+
+export default class TimeConverter extends React.Component {
+
+    constructor(props){
+        super(props);
+        this.state = { hours : 1, minutes: 60, seconds: 3600}
+    }
+
+    hourUpdate = (e) => {
+        const val = this.minutes.value;
+        this.setState(oldState => ({ hours: val/60, minutes : val, seconds: val * 60 }));
+    }
+
+    minUpdate = (e) => {
+        const val = this.hours.value;
+        this.setState(oldState => ({ hours: val, minutes : val *60, seconds: val * 3600 }));
+    }
+
+    secUpdate = (e) => {
+        const val = this.seconds.value;
+        this.setState(oldState => ({ hours: val/3600, minutes : val/60, seconds: val }));
+    }
+
+
+    render(){
+        var boxStyle = {marginRight : '5px', width: '40px'}
```

Comment:
var? I thought we're all `let` and `const` by now :)
But really `var` keyword should be avoided. If you're curious why you'll probably find this one an interesting read: https://hackernoon.com/why-you-shouldnt-use-var-anymore-f109a58b9b70